jetson/percep/main.cpp

```diff
@@ -7,59 +7,59 @@
 using namespace cv;
 using namespace std;
```

Comment:
would you mind getting rid of this? bad practice

Reply:
yeah it was in the original code but I can remove the `using namespace`s

Reply:
thx